### PR TITLE
docs: remove internal docs and deployment variables 

### PR DESCRIPTION
**Pull request by camargo**
## Description

- Update Environment.md with external deployment vars
- Update docker-compose deployment to have CAM auth OFF by default
- Internal docs are now here: https://github.jpl.nasa.gov/Aerie/aerie-internal-docs
- Add warning to change default passwords to deployment README.md
- Formatting and alphabetization

## Documentation

This PR is primarily documentation updates.

### CODE REVIEW

**Review by pcrosemurgy — APPROVED** [review]

File: deployment/SSL.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# SSL
+
+Some Aerie service configurations require connecting to internal JPL services (e.g. authentication via CAM). For this to work you need to add the proper internal JPL certificates to some of the Aerie containers.
```
Comment (patkenneally) [inline deployment/SSL.md]: While this information is really important to capture and communicate to users, should we be placing this information in the public repo/part of the public docs? I'm pondering our other options. Maybe a separate set of docs specific for JPL users which dovetails with the public docs. 🤔

Comment (camargo) [reply deployment/SSL.md]: Yeah I agree. The deployment docs seemed like the most logical place for now.

Comment (camargo) [reply deployment/SSL.md]: We could put them in a folder `deployment/internal` to separate the docs. I can't think of a simple way to do this though without keeping a completely separate internal document set. 
Also related to this, `cam` auth is the default for our deployment, which implies internal deployment. Maybe we switch the default to `none` and do keep an internal document set?

**Review by patkenneally — APPROVED** [review]

File: deployment/SSL.md (hunk 1 above)
Comment (patkenneally) [reply deployment/SSL.md]: We resolved to move specific deployment concerns to a JPL internal docs repo therefore keeping the main project here generic in application.

**Pull request merged**